styleguide/source/_patterns/01-molecules/06-components/20-testimonial/_testimonial.scss

```diff
@@ -0,0 +1,64 @@
+.testimonial {
+  position: relative;
+  width: 100%;
+  max-width: 1200px;
+  margin-bottom: 24px;
+
+  @include breakpoint($bp-med) {
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: stretch;
+
+    &.testimonial-left {
+      flex-direction: row;
+    }
+  }
+}
+
+.testimonial_image {
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  min-height: calc(((100vw - 40px) / 3) * 2);
+
+  @include breakpoint($bp-med) {
+    min-height: calc(((100vw -  60px) / 2) / 3 * 2);
```

Comment:
I know we're trying to preserve the aspect ratio of the image, but at some in-between sizes, I wonder if this is the behavior we want. Could we maybe wrap the image in a container and leverage our grid to prevent the image cropping in these cases and instead have it scale to fit? Feel free to argue if you think that's not the preferable behavior.

![screen shot 2017-06-21 at 1 35 43 pm](https://user-images.githubusercontent.com/28711067/27402234-9a2cf30c-568b-11e7-8071-797a7a33c71a.png)